ambry-quota/src/main/java/com/github/ambry/quota/MySqlStorageUsageRefresher.java

```diff
@@ -388,6 +390,7 @@ void persistentBackupFile(String filename, Map<String, Map<String, Long>> usage)
       if (backupFiles.contains(filename)) {
         return;
       }
+      logger.trace("Persist container usage for " + filename);
```

Comment:
nit: some remaining string concats in this class

Reply:
updated.